packages/fast-components-react-base/src/dialog/dialog.spec.tsx

```diff
@@ -243,4 +241,26 @@ describe("dialog", (): void => {
         expect(mockRemoveListenerFn).toHaveBeenCalledTimes(2);
         expect(mockRemoveListenerFn.mock.calls[1][0]).toBe("focusin");
     });
+    test("shouldForceFocus correctly detects whether the prodided element is in or out of dialog", () => {
```

Comment:
nit: provided* i would update the wording here to something like "should detect whether the provided element is in or out of dialog"